project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.4.9
```

Comment:
Hmm I'm getting the following when loading the project in sbt, after most dependencies have been resolved:

```
[info] welcome to sbt 1.4.9 (Red Hat, Inc. Java 11.0.8)
[info] loading settings for project global-plugins from plugins.sbt ...
[info] loading global plugins from /home/plipski/.sbt/1.0/plugins
[info] loading project definition from /home/plipski/akka-safer-serializer/project
[error] java.lang.NoSuchMethodError: 'scala.Function1 sbt.Defaults$.foldMappers(scala.collection.Seq)'
[error] 	at sbt.errorssummary.Plugin$.$anonfun$reporterSettings$1(Plugin.scala:39)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error] 	at java.base/java.lang.Thread.run(Thread.java:834)
[error] (Compile / compile / compilerReporter) java.lang.NoSuchMethodError: 'scala.Function1 sbt.Defaults$.foldMappers(scala.collection.Seq)'
[warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
```

Reply:
Have you seen that yourself? 🤔

Reply:
No, while sbt did cause some problems, this one is new to us

Reply:
Hmmm what's your `java -version`?

Reply:
(although TBH it doesn't seem related to Java per se)

Reply:
```
java version "11.0.5" 2019-10-15 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.5+10-LTS)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.5+10-LTS, mixed mode)
```

Reply:
Ok, this was an issue with a global sbt plugin I was using (`addSbtPlugin("org.duhemm" % "sbt-errors-summary" % "0.6.3")`)... so nvm, no action required